lib/fluent/plugin/out_sumologic.rb

```diff
@@ -213,6 +235,10 @@ def write(chunk)
       key           = sumo_key(sumo_metadata, chunk)
       log_format    = sumo_metadata['log_format'] || @log_format
 
+      if log_format.eql? 'fields'
+        log_fields    = sumo_fields(sumo_metadata)
```

Comment:
I think we could include this line somewhere in the `when 'fields'` case below (line 258) rather than adding another if statement

Reply:
good catch, will fix